xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -169,6 +169,14 @@ namespace XBMCAddon
       item->GetVideoInfoTag()->SetRating(rating, votes, type, defaultt);
     }
 
+    void ListItem::AddSeason(int number, std::string name /* = "" */)
+    {
+      if (!item) return;
+
+      LOCKGUI;
```

Comment:
@akva2 Fixed it. thanks. So also SetRatings() and all the setters needs this? because I saw some other that didn't have it